baseline.yaml

```
@@ -649,6 +649,27 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+- id: OSPS-DO-14
```

```suggestion
  - id: OSPS-DO-14
```

After you rebase, this will fix the CI error.